src/libs/nav2_robot/include/nav2_robot/ros_robot.hpp

```diff
@@ -37,11 +38,12 @@ class RosRobot : public Robot
 
   void enterSafeState() override;
 
+  urdf::Model model_;
```

Comment:
This should be a protected member variable. There should be public methods used to access the data held in the model. For example, you could add a getFootprint() method that other code could use to get the robot footprint.

Reply:
Agreed. 
I'm looking into footprint to see what we can do.